### PR TITLE
add kibana logs for all job

### PR DESCRIPTION
**Pull request by bhaskarhc**
Screenshot from 2019-08-23 15-22-20

Signed-off-by: bhaskarjb <hcbhaskar7@gmail.com>

### CODE REVIEW

File: database/database.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ func createTable() {
 	}
 	// Create pipeline jobs table in database
 	for i := range pipelineJobs {
-		query := fmt.Sprintf("CREATE TABLE IF NOT EXISTS %s(pipelineid INT, id INT PRIMARY KEY,status VARCHAR, stage VARCHAR, name VARCHAR, ref VARCHAR, created_at VARCHAR, started_at VARCHAR, finished_at VARCHAR);", pipelineJobs[i])
+		query := fmt.Sprintf("CREATE TABLE IF NOT EXISTS %s(pipelineid INT, id INT PRIMARY KEY,status VARCHAR, stage VARCHAR, name VARCHAR, ref VARCHAR, created_at VARCHAR, started_at VARCHAR, finished_at VARCHAR, job_kibana_url VARCHAR);", pipelineJobs[i])
```
Comment (chandankumar4) [inline database/database.go]: Use variable name `job_log_url`

File: handler/types.go [hunk 2]
```diff
@@ -70,6 +70,7 @@ type Jobssummary struct {
 	CreatedAt  string `json:"created_at"`
 	StartedAt  string `json:"started_at"`
 	FinishedAt string `json:"finished_at"`
+	JobLogURL  string `json:"job_kibana_url"`
```
Comment (chandankumar4) [inline handler/types.go]: JobLogURL  string `json:"job_kibana_url"` ->  LogURL  string `json:"job_log_url"`

File: handler/packet.go [hunk 3]
```diff
@@ -116,9 +116,11 @@ func PacketData(token, triggredIDColumnName, pipelineTableName, jobTableName str
 		glog.Infoln("New record ID for PACKET Pipeline:", id)
 		if pipelinedata.ID != 0 {
 			for j := range pipelineJobsdata {
-				sqlStatement := fmt.Sprintf("INSERT INTO %s (pipelineid, id, status, stage, name, ref, created_at, started_at, finished_at) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)"+
-					"ON CONFLICT (id) DO UPDATE SET status = $3, stage = $4, name = $5, ref = $6, created_at = $7, started_at = $8, finished_at = $9 RETURNING id;", jobTableName)
+				var jobLogURL string
+				sqlStatement := fmt.Sprintf("INSERT INTO %s (pipelineid, id, status, stage, name, ref, created_at, started_at, finished_at, job_kibana_url) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10)"+
+					"ON CONFLICT (id) DO UPDATE SET status = $3, stage = $4, name = $5, ref = $6, created_at = $7, started_at = $8, finished_at = $9, job_kibana_url = $10 RETURNING id;", jobTableName)
 				id := 0
+				jobLogURL = Kibanaloglink(packetPipelineData.Sha, packetPipelineData.ID, packetPipelineData.Status, pipelineJobsdata[j].StartedAt, pipelineJobsdata[j].FinishedAt)
```
Comment (chandankumar4) [inline handler/packet.go]:
```
if packetPipelineData.ID != 0 && len(pipelineJobsdata) != 0 {
jobLogURL = Kibanaloglink(packetPipelineData.Sha, packetPipelineData.ID, packetPipelineData.Status, pipelineJobsdata[j].StartedAt, pipelineJobsdata[j].FinishedAt)
}
```
Change the above line

Comment (bhaskarhc) [reply handler/packet.go]: @chandankumar4  already check is there to get jobs , here also required ???

File: handler/konvoy.go [hunk 4]
```diff
@@ -143,9 +144,13 @@ func KonvoyData(token, triggredIDColumnName, pipelineTableName, jobTableName str
 		glog.Infoln("New record ID for KONVOY Pipeline:", id)
 		if pipelinedata.ID != 0 {
 			for j := range pipelineJobsdata {
-				sqlStatement := fmt.Sprintf("INSERT INTO %s (pipelineid, id, status, stage, name, ref, created_at, started_at, finished_at) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)"+
-					"ON CONFLICT (id) DO UPDATE SET status = $3, stage = $4, name = $5, ref = $6, created_at = $7, started_at = $8, finished_at = $9 RETURNING id;", jobTableName)
+				var jobLogURL string
+				sqlStatement := fmt.Sprintf("INSERT INTO %s (pipelineid, id, status, stage, name, ref, created_at, started_at, finished_at, job_log_url) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10)"+
+					"ON CONFLICT (id) DO UPDATE SET status = $3, stage = $4, name = $5, ref = $6, created_at = $7, started_at = $8, finished_at = $9, job_log_url = $10 RETURNING id;", jobTableName)
 				id := 0
+				if pipelinedata.ID != 0 && len(pipelineJobsdata) != 0 {
```
Comment (chandankumar4) [inline handler/konvoy.go]: Can you remove the check of pipelinedata.id != 0, it's already on the line no 145

File: handler/konvoy.go [hunk 5]
```diff
@@ -143,9 +144,13 @@ func KonvoyData(token, triggredIDColumnName, pipelineTableName, jobTableName str
...
+				jobLogURL = Kibanaloglink(konvoyPipelineData.Sha, konvoyPipelineData.ID, konvoyPipelineData.Status, pipelineJobsdata[j].StartedAt, pipelineJobsdata[j].FinishedAt)
```
Comment (chandankumar4) [inline handler/konvoy.go]: Here is some indentation issue

File: handler/packet.go (hunk 3 above)
Comment (chandankumar4) [reply handler/packet.go]: Not required, please fix the indentation issue

**Review by chandankumar4 — CHANGES_REQUESTED** [review]

**Review by chandankumar4 — APPROVED** [review]
lgtm

**Pull request merged**